crates/sdk/src/wallet/store.rs

```diff
@@ -654,6 +655,13 @@ impl Store {
         ];
         self.pkhs.extend(pkhs);
 
+        self.addresses
+            .extend([(address_alias.clone(), validator_address.clone())]);
+        println!(
+            "\nAdded the validator address {} to the wallet with alias \"{}\".",
+            &validator_address, address_alias
+        );
```

Comment:
the `println!()` can remain in `crates/apps/src/lib/client/utils.rs`

Reply:
done in [03006d8](https://github.com/anoma/namada/pull/3093/commits/03006d891727ec1a624b55cdb8d1fa85ab227347)

Reply:
Actually would like to keep this here, since the validator address alias is in scope here but not in the previous location yet. Seems more consistent to keep it here to me, but if I'm open to reconsidering if you felt more strongly about the other option.

Reply:
There are unfortunately already some "naked" prints in this module (and probably elsewhere too), but ideally in the SDK crate they should only be using the `(e)display_line` macro, for which we'd need the generic `sdk::io::Io` in scope which would probably be the best option to enable us to replace the other ones too. I'd say for this PR, one more in here won't make much difference, so I'll open an issue to address this separately

Reply:
#3106